create_student.rb

```diff
@@ -18,5 +20,6 @@ def create_student
 
   student = Student.new(age, nil, name, parent_permission: permission)
   @people.push(student)
+  save_student(age, name, permission)
```

Comment:
- [ ] you are saving to the json file each time the student gets created, the issue with this is that after adding the student and then showing the list of students the added student will appear twice. you can solve this by updating all the json files when the user is closing the app.